src/gei/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -396,7 +396,7 @@ private void ValidateGHESOptions(MigrateRepoCommandArgs args, bool cloudCredenti
 
             if (args.UseGithubStorage)
             {
-                _log.LogWarning("Ignoring --use-github-storage flag because you are running GitHub Enterprise Server (GHES) 3.8.0 or later. The blob storage credentials configured in your GHES Management Console will be used instead.");
+                _log.LogWarning("Provding the --use-github-storage flag will supersede any credentials you have configured in your GitHub Enterprise Server (GHES) Management Console.");
```

Comment:
as described [here](https://github.com/github/gh-gei/pull/1293#discussion_r1838631595) please test this logic to confirm that it's the case.

Reply:
Manually tested and it is the case